sdk/storage/Azure.Storage.DataMovement.Blobs/README.md

```diff
@@ -45,189 +45,195 @@ The Azure.Storage.DataMovement.Blobs library uses clients from the Azure.Storage
     sourceResource: new LocalFileStorageResource(sourceLocalPath),
     destinationResource: new BlockBlobStorageResource(destinationBlob));
 await dataTransfer.AwaitCompletion();
 ```
-Apply Options to Block Blob Download
-```C# Snippet:BlockBlobDownloadOptions
-await transferManager.StartTransferAsync(
-    sourceResource: new BlockBlobStorageResource(sourceBlob, new BlockBlobStorageResourceOptions()
-    {
-        DestinationConditions = new BlobRequestConditions(){ LeaseId = "xyz" }
-    }),
-    destinationResource: new LocalFileStorageResource(downloadPath2));
-```
 
-Start Directory Upload
-```C# Snippet:SimpleDirectoryUpload
-// Create simple transfer directory upload job which uploads the directory and the contents of that directory
+Upload a directory as a specific blob type.
+
+```csharp
+string sourceLocalPath;
+BlobContainerClient destinationContainer;
+string optionalDestinationPrefix;
+
 DataTransfer dataTransfer = await transferManager.StartTransferAsync(
-    sourceResource: new LocalDirectoryStorageResourceContainer(sourcePath),
+    sourceResource: new LocalDirectoryStorageResourceContainer(sourceLocalPath),
     destinationResource: new BlobStorageResourceContainer(
-        container,
-        new BlobStorageResourceContainerOptions() { DirectoryPrefix = "sample-directory2" }),
-    transferOptions: options);
+        destinationContainer,
+        new BlobStorageResourceContainerOptions()
+        {
+            // Block blobs are the default if not specified
+            BlobType = BlobType.Block,
+            DirectoryPrefix = optionalDestinationPrefix
+        }));
+await dataTransfer.AwaitCompletion();
+```
+
+### Download
+
+A download takes place between a blob `StorageResource` as source and local file `StorageResource` as destination.
+
+Download a block blob.
+
+```csharp
+BlockBlobClient sourceBlob;
+string destinationLocalPath;
+
+DataTransfer dataTransfer = await transferManager.StartTransferAsync(
+    sourceResource: new BlockBlobStorageResource(sourceBlob),
+    destinationResource: new LocalFileStorageResource(destinationLocalPath));
+await dataTransfer.AwaitCompletion();
 ```
 
-Start Directory Download
-```C# Snippet:SimpleDirectoryDownload
-DataTransfer downloadDirectoryJobId2 = await transferManager.StartTransferAsync(
-    sourceDirectory2,
-    destinationDirectory2);
+Download a container which may contain a mix of blob types.
+
+```csharp
+BlobContainerClient sourceContainer;
+string optionalSourcePrefix;
+string destinationLocalPath;
+
+DataTransfer dataTransfer = await transferManager.StartTransferAsync(
+    sourceResource: new BlobStorageResourceContainer(
+        sourceContainer,
+        new BlobStorageResourceContainerOptions()
+        {
+            DirectoryPrefix = optionalSourcePrefix
+        }),
+    destinationResource: new LocalDirectoryStorageResourceContainer(destinationLocalPath));
+await dataTransfer.AwaitCompletion();
 ```
 
-Simple Logger Sample for Transfer Manager Options
```

Comment:
It looks like I've moved that into the base datamovement package since it's on the transfer manager itself. I replaced the exact logging bit with a more generic "handle this event as you please" but we can revert back to the specific logger sample.

Reply:
Sample is now in base package readme and once again using logging as a sample action.